### DISCUSSION TITLE
[Feature request] Spam IDs (invitation codes)

### DISCUSSION THREAD

**Discussion by serrq** [element-hq/element-meta#1921] [Ideas]
I would like an extra layer of protection against spam.

It is true that I can reject a request for connection that I do not like, but it is equally true that I could be targeted by a fleet of spam.

What is the point of rejecting 10000 requests for connection?

For job search purposes I decided to integrate Matrix into my contact details.

However I need to be contacted after the prospective contact has entered an invitation code previously issued on other channels.

The control over that invitation (spam ID) is complete: I can activate, disable, delete an invitation. I can program it to receive only X invitations per day, or receive pairing requests only in the summer period (for example).

Of course I can manage multiple spam id at once, and to give them labels in order to remember me what targets they are hitting.

**Comment by kittykat** [comment]
Hi, thank you for filing this idea! I'm going to move this to Element meta discussions as that is where we keep track of feature requests :+1:
